src/components/text-editor/examples/text-editor-with-inline-images-file-storage.tsx

```diff
@@ -0,0 +1,118 @@
+import { Component, h, State, Host } from '@stencil/core';
+import {
+    ImageInserter,
+    FileInfo,
+    ImageInfo,
+    LimelTextEditorCustomEvent,
+    LimelCheckboxCustomEvent,
+} from '@limetech/lime-elements';
+/**
+ * Handling inline images (with external file storage)
+ *
+ * To allow users to paste images directly into the text editor, you can
+ * listen to the `imagePasted` event, which is triggered when an image file
+ * is pasted into the editor.
+ *
+ * The `imagePasted` event contains an `ImageInserter` object, which you can
+ * use to insert a thumbnail of the pasted image into the editor.
+ * After the thumbnail is inserted, you can upload the image to an external
+ * file storage and insert the src url of the uploaded image using the
+ * `insertImage` method.
+ *
+ * If the image upload fails, you can insert a failed thumbnail using the
+ * `insertFailedThumbnail` method.
+ */
```

Comment:
I pasted an image, and once it was uploaded, it was replaced by the Lime CRM logo. I understand that's because we want to show an image that is served from somewhere, but we should probably add something about that in this description. An INFO admonition perhaps. Something like this:

> [!NOTE]
> In this example, because we don't actually upload the image you paste anywhere, once the "upload" is done, we will replace the image you pasted with a url to an image of the Lime CRM logo. In reality, you would of course insert the url to the newly uploaded image instead 😄

Reply:
⚡ [6f0b8e6](https://github.com/Lundalogik/lime-elements/pull/3464/commits/6f0b8e690487e3dfe080ed97cfc6b08609fbf705)